HimotokiTests/DecodeWithRootKeyPathTest.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  DecodeWithRootKeyPathTest.swift
+//  Himotoki
+//
+//  Created by Syo Ikeda on 11/10/15.
+//  Copyright © 2015 Syo Ikeda. All rights reserved.
+//
+
+import XCTest
+import Himotoki
+
+class DecodeWithRootKeyPathTest: XCTestCase {
+
+    lazy var JSON: [String: AnyObject] = {
```

Comment:
Name Format Violation: Variable name should start with a lowercase character: 'JSON'